### ISSUE TITLE
pylint 1.7.x: false negative "no-member" for RootLogger methods

### ISSUE THREAD

**Issue by chenghlee** [PyCQA/pylint#1585]
### Steps to reproduce
1. Create `rootlogger-no-member.py` script:
```
from __future__ import print_function
import logging

logger = logging.getLogger()
logger.bogus("foo")
```
2. Run `pylint -rn --disable=all --enable=E rootlogger-no-member.py`

### Current behavior
Pylint 1.7.0, 1.7.1, and 1.7.2 fail to recognize that the RootLogger instance (`logger`) does not have a `bogus` member:
```
No config file found, using default configuration

------------------------------------
Your code has been rated at 10.00/10
```

### Expected behavior
Pylint 1.6.5 correctly complains about the missing `RootLogger` member:
```
No config file found, using default configuration
************* Module pylint-logging-test
E:  5, 0: Instance of 'RootLogger' has no 'bogus' member (no-member)
```

### pylint --version output
Fails on both Python 2.7 and 3.5:
```
No config file found, using default configuration
pylint 1.7.2, 
astroid 1.5.3
Python 2.7.13 (default, Jul 18 2017, 09:16:53) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```
---
```
No config file found, using default configuration
pylint 1.7.2, 
astroid 1.5.3
Python 3.5.3 (default, Jun  8 2017, 14:33:16) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```

**Comment by PCManticore** [comment]
Try to pass ``--ignore-on-opaque-inference=n`` to pylint, this will make it complain about the RootLogger. We added this flag in order to control if we should emit this check when we have unknown inference objects when trying to infer the owner of the attribute.
